docs/triggers_rules.md

```diff
@@ -141,3 +142,6 @@ As an example, create a rule that calls the hello action whenever a location upd
   You see that the hello action received the event payload and returned the expected string.
 
   You can create multiple rules that associate the same trigger with different actions.
+
+The trigger and action that make a rule must be in the same namespace, which is in fact private.
```

Comment:
indent.

FYI this constraint should be lifted soon now that new activator is merged.